honestcash-v2/src/app/main/pages/story/story.component.html

```diff
@@ -0,0 +1,11 @@
+
+<div *ngIf="story" class="mb-5 story-preview__wrapper">
+  <main-story-title [story]="story"></main-story-title>
```

Comment:
I'll probably add a story store and make these sub components load story from store